### ISSUE TITLE
Unit on low altitude map not stalling out, making incorrect PSR

### ISSUE THREAD

**Issue by SuperStucco** [MegaMek/megamek#4938]
In the attached 49.12 save, there is a Princess controlled Thunderbird fighter at the north end of the map.  It has taken a single engine hit and due to the large load of rocket pods it effectively has zero thrust.  Thanks to the low altitude rules it has dropped to zero velocity as well.  However, each round it is making a PSR for "Velocity greater than x2 thrust" despite it having both zero thrust and zero velocity:

`[Thunderbird TRB-D36] (Federated Commonwealth Allied Bombers) must make 1 control roll(s) (Velocity greater than 2x safe thrust).
The base target is [5] [4 (Base piloting skill) + 2 (Atmospheric operations) - 1 (fighter/small craft) + 0 (Velocity greater than 2x safe thrust)].
    Roll #1, (Velocity greater than 2x safe thrust); needs 5, rolls 10 : succeeds.`

Even though it is at zero velocity in atmosphere, it is not making a stall check, so every round it is just hanging in mid-air.

[Zero_thrust_vs_stall.sav.gz](https://github.com/MegaMek/megamek/files/13537272/Zero_thrust_vs_stall.sav.gz)

(Advisory - the map for this is VERY large, 77x110)

In short, the PSR for the velocity vs thrust is invalid (probably a >= operator instead of a >).  More generally it seems control rolls seem to get lumped together at times with the result of passing one is treated as passing others.  Individual control rolls like stall checks should be individual, same as with piloting checks on Mechs.

**Comment by neoancient** [comment]
This is specific to Princess. She is sending an illegal movement to the server (ACC, D, F), which rejects it and does not process it.
